src/rest_service/result/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+import uuid
+
+# Create your models here.
+
+class Result(models.Model):
+    id = models.UUIDField(
+        _("Id"),
+        primary_key=True,
+        db_column="result_id",
+        db_index=True,
+        auto_created=True,
+        unique=True,
+        editable=False,
+        default=uuid.uuid4
+    )
+
+    # In team_id ForeignKey cant write 'team.Team.id'
+    # is 'team.Team' ok?
+    # This goes for scenario_id too..
+    team_id = models.ForeignKey('team.Team', on_delete=models.CASCADE)  # should on_delete stayed on CASCADE or change?
+    scenario_id = models.ForeignKey('test.Scenario', on_delete=models.CASCADE) # should on_delete stayed on CASCADE or change?
```

Comment:
first :
    No, we want to delete records of a deleted team or deleted scenario
second:
imagine a code like this
```
scenario = scenario.objects.first()
a = Result()
a.scenario_id = scenario
a.team_id = fooo
a.save()
-----
a = Result()
a.senario = scenario
a.team = fooo
a.save()
```
in the first one you pass a Scenario obj to scenario_id field
as same as team
in the second on you pass a scenario obj to scenario field

you can choose which one to choose  
```suggestion
    team = models.ForeignKey('team.Team', on_delete=models.CASCADE, db_column="team_id") 
    scenario = models.ForeignKey('test.Scenario', on_delete=models.CASCADE, db_column="scenario_id") 
```